coverage/lcov-report/src/migration/1749751059852-rename_tables.ts.html

```
@@ -160,7 +160,7 @@ <h1><a href="../../index.html">All files</a> / <a href="index.html">src/migratio
             <div class='footer quiet pad2 space-top1 center small'>
```

HTMLHint (1.5.0)

[error] The value of attribute [ class ] must be in double quotes.

(attr-value-double-quotes)